src/shared/bootspec.c

```diff
@@ -1811,7 +1843,7 @@ int show_boot_entry(
 
                 const char *p = e->root ? path_startswith(e->path, e->root) : NULL;
                 if (p) {
-                        text = strjoin(ansi_grey(), e->root, "/", ansi_normal(), "/", p);
+                        text = strjoin(ansi_grey(), e->root, "/", ansi_normal(), p);
```

Comment:
This is intentional and double / is printed consistently for paths under $BOOT. See comment in boot_entry_file_list for details. Hence please drop this change